packages/framework/fluid-static/src/index.ts

```diff
@@ -33,3 +32,8 @@ export type {
 	MemberChangedListener,
 	Myself,
 } from "./types.js";
+
+// Re-export so other packages don't need to pull in container-runtime
+// TODO: Should we re-export?
```

Comment:
I don't see a great reason to re-export the API in fluid-static, but I believe this would be a breaking change to a public API. Do we need to stage removing the API completely from fluid-static?